adoc/admin-centralized-logging.adoc

```diff
@@ -1,6 +1,167 @@
+// so that the user doesn't have to scroll through the entire line.
+// Where would it logically make sense to add these?
+
+== Installing Helm and Tiller
+
+In order to successfully use Centralized Logging, you first need to install `Helm` and `Tiller`.
+Helm is used to install the log agents and provide custom logging settings.
+
+- As of {productname} {productversion},
+Helm is not part of the {productname} package repository, so to use Centralized Logging,
+install the {productname} 3 package to install the Helm client:
+
+//TODO: where do I have to be to run these commands?
+
+[source,bash]
+----
+sudo zypper addrepo http://download.suse.de/ibs/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/SUSE:Updates:SUSE-CAASP:3.0:x86_64.repo
+sudo zypper refresh
+sudo zypper install -y helm
+
+helm init --client-only
+----
+
+The official Helm installation can also be used to setup the Helm client and server side.
+
+//TODO: link to official docs or provide specific steps.
+
+////
+Note: what follows here are the instutions to be included after the Helm package has been integrated into v4:
+To install Helm, use the common zypper command:
+----
+sudo zypper install helm
+----
+
+Then initialize the Helm client by running:
+----
+helm init --client-only
+----
+////
+
+
+- As of {productname} {productversion},
+Tiller is not part of the {productname} package repository,
+so to install the {productname} 3 Tiller container image:
+
+[source,bash]
+----
+kubectl create serviceaccount --namespace kube-system tiller
+kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+
+helm init --tiller-image registry.suse.com/sles12/tiller:2.8.2 --service-account tiller
+----
+
+//TODO: What do we expect to happen here? Expected output after `helm init ...`
+
+////
+Note: When Helm is included in v4, Tiller server will be automatically installed after CaaS Platform setup.
+So we probably  just need to mention that we use it and that it's installed automatically.
+////
+
+== Deployment
+
+After you have successfully installed it,
+use Helm CLI to install log agents on each node,
+and provide customized settings via specific command options.
+
+The only three mandatory parameters for a successful deployment of Centralized Logging
+are:
+
+* `*server.host*`, default value = rsyslog-server.default.svc.cluster.local
+* `*server.port*`, default value = 514
+* `*server.protocol*`, default value = TCP
+
+See <<Optional settings>> for the facultative parameters and their default values.
+
+- Running the following will create the minimal working setup:
+
+[source,bash]
+----
+helm repo add suse-charts https://kubernetes-charts.suse.com
+helm install stable/log-agent-rsyslog --name ${RELEASE_NAME} --set server.host=${SERVER_HOST} --set server.port=${SERVER_PORT}
+----
+
+//TODO: one of the three mandatory settings is the server.protocol,
+//but it's not set above. Where is it set?
+
```

Comment:
If it is not include, it will use the default value `TCP`.  We could add --set server.protocol=${SERVER_PROTOCOL} to helm install for clarification.

Reply:
I would just add that it's set to TCP by default and that should be enough.